### PR TITLE
feat(utxo-bin): add command `parseAddress`

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
Add address parsing to utxo-bin.

Features:

- script type detection
- conversion to other networks

Issue: BG-60608

### CODE REVIEW

File: modules/utxo-bin/test/fixtures/formatAddress_361UWADXqgPsdf8L5Jga6GNPauMuBtT1eW.all.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  ├── bitcoinsv: 361UWADXqgPsdf8L5Jga6GNPauMuBtT1eW
+  ├── bitcoinsvTestnet: 2MwZgZu9ZT8uDqSkskSJSiDMeoFa4z4RtdW
+  ├── dash: 7Wj7LMVhefVWdRfHLvM5ReN3WTCGzyVbdE
+  ├── dashTest: 8ijvHgPZnCt95j5YRBM2t2BQPxy76bvp6E
+  ├── dogecoin: 9vkjF1HRukGmY2VoVSLzLPzmHUjwCvtAad
+  ├── dogecoinTest: 2MwZgZu9ZT8uDqSkskSJSiDMeoFa4z4RtdW
+  ├── litecoin: MCDcp3dVnoFJSAQEBBfuuucnubxM6uQRD5
+  ├── litecoinTest: QQvSgv1oUExJydWvNYLTnuo5we1tsV5DZ3
+  ├── zcash: t3Nt5WVdfp1BUEJBE1jVhE5UJqZYz1DCoRU
+  └── zcashTest: t2AsGZAjogTnqmzkxwUVjmhewx3n9qwJkTV
```
Comment (OttoAllmendinger) [inline modules/utxo-bin/test/fixtures/formatAddress_361UWADXqgPsdf8L5Jga6GNPauMuBtT1eW.all.txt:27]: sample output for `parseAddress 361UWADXqgPsdf8L5Jga6GNPauMuBtT1eW --all`

File: modules/utxo-bin/src/commands.ts [hunk 2]
```diff
@@ -100,19 +142,32 @@ export const cmdParse = {
...
+
+export const cmdParseAddress = {
+  command: 'parseAddress [address]',
+  aliases: ['address'],
+  describe: 'parse address',
+  builder(b: yargs.Argv<unknown>): yargs.Argv<ArgsParseAddress> {
+    return b
+      .option('network', { alias: 'n', type: 'string' })
+      .option('format', { choices: ['tree', 'json'], default: 'tree' } as const)
+      .option('convert', { type: 'boolean', default: false })
```
Comment (sangaman) [inline modules/utxo-bin/src/commands.ts:164]: Did you consider adding aliases for these other options?

**Review by sangaman — CHANGES_REQUESTED** [review]
Great test fixture coverage. I didn't try running this tool myself but the code LGTM.

**Review by sangaman — APPROVED** [review]

File: modules/utxo-bin/src/commands.ts (hunk 2 above)
Comment (OttoAllmendinger) [reply modules/utxo-bin/src/commands.ts:164]: you mean like single letter aliases? I'll add them when the long options get inconvenient for me

**Pull request merged**